cfme/intelligence/reports/reports.py

```diff
@@ -190,16 +176,10 @@ class ReportTimelineView(CloudIntelTimelinesView):
 
     @property
     def is_displayed(self):
-        expected_tree = [
-            self.context["object"].type or self.context["object"].company_name,
-            self.context["object"].subtype or "Custom",
-            self.context["object"].menu_name
-        ]
-
         return (
             self.logged_in_as_current_user and
             self.navigation.currently_selected == ['Cloud Intel', 'Timelines'] and
-            self.timelines.tree.currently_selected == expected_tree
+            self.timelines.tree.currently_selected == self.context["object"].tree_path[1:]
```

Comment:
Optional: A small comment here to explain why you're doing `tree_path[1:]` may be helpful.